app/screens/find_channels/utils.ts

```diff
@@ -25,3 +25,12 @@ export const retrieveChannels = (database: Database, myChannels: MyChannelModel[
 
     return of$([]);
 };
+
+export const removeChannelsFromArchivedTeams = (recentChannels: ChannelModel[], teamIds: Set<string>) => {
```

Comment:
this is fine as deleted teams should not really be in the database. But I wonder if this function should also check that the `team.deleteAt` is equal to 0 as a safety net.

Reply:
I initially was trying to add the deleteAt property to Team model, but then realized the existing teams were always the active ones so there was no need to add the deleteAt property to team model.